tests/idv_service/session/retrieve/allowed.source.response.spec.js

```diff
@@ -0,0 +1,22 @@
+const AllowedSourceResponse = require('../../../../src/idv_service/session/retrieve/allowed.source.response');
```

Comment:
Hum...
We already have a class called `allowed.source.response.js` in `/src/idv_service/session/retrieve/allowed.source.response.js` , so why are we introducing a new one ?